lib/vtls/darwinssl.c

```diff
@@ -364,7 +365,9 @@ CF_INLINE const char *SSLCipherNameForNumber(SSLCipherSuite cipher)
   }
   return "SSL_NULL_WITH_NULL_NULL";
 }
+#endif /*  CURL_DISABLE_VERBOSE_STRINGS */
```

Comment:
I think you can remove lines 368 and 370, so both inline functions are under the same preprocessor check. Otherwise, this looks fine to me.